examples/game_of_tag/game_of_tag.py

```diff
@@ -0,0 +1,282 @@
+"""Let's play tag!
```

Comment:
I do not think it is necessary to have the python example anymore if we already have the working notebook version.